commons/servers/metrics.ts

```diff
@@ -37,12 +37,12 @@ export class Metrics {
 
   private static requestCounters = function (req, _res, next) {
     numRequests.inc({ method: req.method })
-    pathsTaken.inc({ path: req.route.path })
+    pathsTaken.inc({ path: req.baseUrl + req.route.path })
     next()
   }
 
   private static responseCounters = ResponseTime(function (req, res, time) {
-    responses.labels(req.method, req.route.path, res.statusCode).observe(time)
+    responses.labels(req.method, req.baseUrl + req.route.path, res.statusCode).observe(time)
```

Comment:
What does this do?

Reply:
In most cases (like comms or content server) it does nothing. But in lambdas, we registered a router like this

```
this.app.use('/collections', router...)
```

And inside the router, we did:
```
router.get('/wearables', (req, res) => ...)
```

So `req.route.path` would only return '/wearables', but adding the baseUrl makes it contain '/collections also